content/blog/blog-post-10.md

```diff
@@ -0,0 +1,286 @@
+---
+authors: ["kokkos-team"]
+title: "kokkos-fft: FFT interface for Kokkos eco-system"
+date: 2025-03-15
+tags: ["blog"]
+thumbnail: img/blog/2025/kokkos-fft/kokkos-fft.png
+---
+
+# Introduction
+
+The fast Fourier transform (FFT) is a family of fundamental algorithms that is widely used in scientific computing and other areas [1]. [kokkos-fft](https://github.com/kokkos/kokkos-fft) is designed to help [Kokkos](https://github.com/kokkos/kokkos) users who are:
+
+* using Kokkos to port a legacy application which relies on FFT libraries. E.g., fluid simulation codes with periodic boundaries, plasma turbulence, etc.
+
+* wishing to integrate in-situ signal and image processing with FFTs. E.g., spectral analyses, low pass filtering, etc.
+
+* __NOT__ willing to go through the documentation of de facto standard FFT libraries. They want to benefit from powerful vendor FFT libraries while working with a simple API like that of [`numpy.fft`](https://numpy.org/doc/stable/reference/routines.fft.html).
```

Comment:
I don't get that point.
Is that about saying that the kokkos-fft API is easier to use than other C++ alternatives and inspired from numpy.fft?

Reply:
I think that "easier" point is kind of subjective. It is certainly more intuitive to use for Python users, but I am not sure it is the case for hardcore fftw Fortran users.

Reply:
Maybe "intuitive" then :)